### PR TITLE
[feat]: [CDS-36683]: NG Artifactory Trigger changes

### PR DESCRIPTION
**Pull request by piyush-bhuwalka**
You can use the following comments to re-trigger PR Checks

- Compile: `trigger compile`
- runAeriformCheck: `trigger AeriformCheck`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/33171)

### CODE REVIEW

**Comment by piyush-bhuwalka** [comment]
trigger feature-build

**Comment by piyush-bhuwalka** [comment]
trigger feature-build

**Review by inderschera — DISMISSED** [review]
> Reviewed 10 of 20 files at r1, 10 of 10 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/33171)* status: :shipit: complete! ✓

File: 867-polling-contracts/src/main/proto/io/harness/polling/contracts/polling_payload_data.proto [hunk 1]
```diff
@@ -83,8 +84,15 @@ message Nexus3RegistryPayload {
...
-  string artifactDirectory = 3;
-  string repositoryFormat = 4;
+  string repository = 1;
+  string artifactDirectory = 2;
+  string repositoryFormat = 3;
+  string artifactPath = 4;
+  string repositoryUrl = 5;
+}
+
+message ArtifactoryGenericRegistryPayload {
```
Comment (sainath-b) [inline 867-polling-contracts/src/main/proto/io/harness/polling/contracts/polling_payload_data.proto:94]: is it required?

**Review by sainath-b — DISMISSED** [review]

**Comment by piyush-bhuwalka** [comment]
trigger feature-build

File: 867-polling-contracts/src/main/proto/io/harness/polling/contracts/polling_payload_data.proto (hunk 1 above)
Comment (piyush-bhuwalka) [reply 867-polling-contracts/src/main/proto/io/harness/polling/contracts/polling_payload_data.proto]: removed

Comment (piyush-bhuwalka) [reply 867-polling-contracts/src/main/proto/io/harness/polling/contracts/polling_payload_data.proto]: removed

**Review by sainath-b — APPROVED** [review]

**Review by sainath-b — APPROVED** [review]
> Reviewed 9 of 20 files at r1, 9 of 10 files at r2, 5 of 5 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/33171)* status: :shipit: complete! ✓

**Pull request merged**